tasks.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""Read the Docs invoke tasks."""
+
+from __future__ import division, print_function, unicode_literals
+
+import os
+import textwrap
+
+from future.moves.configparser import RawConfigParser
+from invoke import task
+
+
+@task
+def prepare(ctx, version):
+    """
+    Prepare the next release version by updating files.
+
+    This will stage a few updates for manual review and commit:
+
+    * Prepend the most recent PRs and issues that were closed to CHANGELOG.md.
+    * Update the setup.cfg version
+
+    Changelog uses the file modification date to track the last time it was
+    updated.  New entries will end up at the top of the file, under a heading
+    for the new version.
+    """
+    print('Updating release version in setup.cfg')
+    setupcfg_path = os.path.join(os.path.dirname(__file__), 'setup.cfg')
+    config = RawConfigParser()
+    config.read(setupcfg_path)
+    config.set('metadata', 'version', version)
+    with open(setupcfg_path, 'wb') as configfile:
+        config.write(configfile)
```

Comment:
Isn't this the other way around?

    configfile.write(config)

Reply:
Hah, that would make sense though. :)

The ConfigParser API is goofy:
https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.write